### PR TITLE
Add `extend-immutable-calls` setting for B008

### PR DESCRIPTION
**Pull request by edgarrmondragon**
Part of #389

Example:

```toml
[tool.ruff.flake8-bugbear]
extend-immutable-calls = ["fastapi.Depends", "fastapi.Query"]
```

### CODE REVIEW

File: resources/test/fixtures/B008_extended.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+def this_is_okay_extended(db=fastapi.Depends(get_db)):
+    ...
+
+
+def this_is_okay_extended_second(data: List[str] = fastapi.Query(None)):
+    ...
+
+
+def this_is_not_okay_relative_import_not_listed(data: List[str] = Query(None)):
```
Comment (charliermarsh) [inline resources/test/fixtures/B008_extended.py:15]: I feel like this should work / be considered ok. What do you think? (It doesn’t have to be part of this PR, just wondering if it’s _desired_ that it’s not accepted.)

Comment (charliermarsh) [reply resources/test/fixtures/B008_extended.py:15]: (I want to refactor some of the import-from tracking anyway.)

Comment (edgarrmondragon) [reply resources/test/fixtures/B008_extended.py:16]: I copied this from the flake8-bugbear repo so the name is probably misleading 🤔

Comment (JonathanPlasse) [reply resources/test/fixtures/B008_extended.py:16]: `convert_exit_to_sys_exit.get_member_import_name_alias()` could for this.
I can look into it.

**Pull request merged**